tests/payload_tools/fitness.py

```
@@ -0,0 +1,49 @@
+        "workout_id": workout_id,
+        "intensity_measure": intensity_measure,
+        "intensity_value": intensity_value,
+        "estimated_calories_per_unit_frequency": estimated_calories_per_unit_frequency
+    }
+
+def build_create_plan_payload(
+    workout_activity_id: int,
+    strata_name: str = "Day 1: Arms",
+    planned_reps: int = 12,
+    planned_sets: int = 3,
+    planned_duration: int = None
```

Ruff (0.15.10)

[warning] PEP 484 prohibits implicit `Optional`

Convert to `T | None`

(RUF013)